packages/framework/src/Model/Order/Item/OrderItemDataFactory.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Model\Order\Item;
+
+class OrderItemDataFactory implements OrderItemDataFactoryInterface
```

Comment:
So You don't think they should be related?

Because for me, I don't see any reason - every factory implements an interface...
I had another idea - one `OrderItemDataFactoryInterface` for all three types - there would be `createItem`, `createTransportItem`, ... but in the end I decided to keep consistency - one data object -> one factory.

Reply:
Yes, I do agree with you, it was just some gut feeling ("objects are in relation -> should their factories be in relation, too?") but otherwise I do not see any reason for that, too. I just wanted to discuss that feeling :smile: